santaProject/YBR/Class/Mountain.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Mountain.swift
+//  santaProject
+//
+//  Created by yeongbinRo on 2021/06/05.
+//
+
+import Foundation
+import MapKit
+
+class Mountain: NSObject,MKAnnotation {
+    
+    var name: String?
+    var peak: String?
+    var height: Double?
+    var depth1: String?
+    var depth2: String?
```

Comment:
```suggestion
    var region: String?
    var subRegion: String?
```
이렇게 바꾸면 의미를 명확하게 알 수 있을 것 같아요

Reply:
네 급하게 만드느라 적절한 네이밍을 생각하지 못했습니다. 요건 의견반영하여 수정하겠습니다.